weather.gismeteo/default.py

```diff
@@ -0,0 +1,769 @@
+# -*- coding: utf-8 -*-
+import os
+import time
+
+import xbmc
+import xbmcgui
+
+from resources.lib.gismeteo import Gismeteo
+from resources.lib.utilities import *
+
+_ = weather.initialize_gettext()
+
+MAX_DAYS      = 7
+MAX_DAILYS    = 7
+MAX_LOCATIONS = 5
+MAX_HOURLY    = 16
+MAX_36HOUR    = 3
+MAX_WEEKENDS  = 2
+
+DATEFORMAT     = weather.date_format
+TIMEFORMAT     = weather.time_format
+
+WEATHER_ICON   = weather.weather_icon
+
+KODILANGUAGE   = xbmc.getLanguage().lower()
+
+CURRENT_TIME = {'unix': time.time()}
+
+CACHE_DIR = os.path.join(xbmc.translatePath('special://temp'), weather.id)
```

Comment:
Addons are allowed to store data only in their profile directories. A profile dir can be obtained via `addon.getAddonInfo('profile')` or `addon.config_dir` in SimplePlugin.